ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -30,6 +30,8 @@ export default class ExperimentalTab extends PureComponent {
     openSeaEnabled: PropTypes.bool,
     transactionSecurityCheckEnabled: PropTypes.bool,
     setTransactionSecurityCheckEnabled: PropTypes.func,
+    securityAlertsEnabled: PropTypes.bool,
+    setSecurityAlertsEnabled: PropTypes.func,
```

Comment:
What if we rename this to be more explicit to the Blockaid toggle? e.g. `securityAlertBlockaidEnabled` 

A reason I bring this up is that we mention to the user "More providers coming soon", which solidifies the objective to scale. This implies we will either need to rename this now or later.